Core/OfficeDevPnP.Core/AppModelExtensions/FieldAndContentTypeExtensions.cs

```diff
@@ -739,6 +739,10 @@ public static void AddFieldToContentTypeByName(this Web web, string contentTypeN
         public static void AddFieldToContentType(this Web web, ContentType contentType, Field field, bool required = false, bool hidden = false)
         {
             contentType.EnsureProperties(c => c.Id, c => c.FieldLinks, c => c.SchemaXml);
```

Comment:
@SchauDK That would be very nice indeed - but I think the whole procedure could be done differently. Shouldn't be neccessary to load the FieldLinks for the content type for each file you want to add.